src/components/Facility/PatientConsultationNotesList.tsx

```diff
@@ -24,80 +25,67 @@ interface PatientNotesProps {
   setReplyTo?: (value: PatientNotesModel | undefined) => void;
 }
 
-const pageSize = RESULTS_PER_PAGE_LIMIT;
-
 const PatientConsultationNotesList = (props: PatientNotesProps) => {
   const {
     state,
     setState,
-    reload,
     setReload,
     disableEdit,
     thread,
     setReplyTo,
+    reload,
   } = props;
-  const consultationId = useSlug("consultation") ?? "";
-
-  const [isLoading, setIsLoading] = useState(true);
+  const queryClient = useQueryClient();
 
-  const fetchNotes = async () => {
-    setIsLoading(true);
-
-    const { data } = await request(routes.getPatientNotes, {
-      pathParams: {
-        patientId: props.state.patientId || "",
-      },
-      query: {
-        consultation: consultationId,
-        thread,
-        offset: (state.cPage - 1) * RESULTS_PER_PAGE_LIMIT,
-      },
-    });
+  const consultationId = useSlug("consultation") ?? "";
 
-    if (data) {
-      if (state.cPage === 1) {
-        setState((prevState) => ({
-          ...prevState,
-          notes: data.results,
-          totalPages: Math.ceil(data.count / pageSize),
-        }));
-      } else {
-        setState((prevState) => ({
-          ...prevState,
-          notes: [...prevState.notes, ...data.results],
-          totalPages: Math.ceil(data.count / pageSize),
-        }));
+  const { data, isLoading, fetchNextPage, hasNextPage } = useInfiniteQuery({
+    queryKey: ["notes"],
+    queryFn: async ({ pageParam = 0 }) => {
+      const response = await request(routes.getPatientNotes, {
+        pathParams: { patientId: state.patientId! },
+        query: { thread, offset: pageParam, consultation: consultationId },
+      });
+      setReload?.(false);
+      return {
+        results: response?.data?.results ?? [],
+        nextPage: pageParam + RESULTS_PER_PAGE_LIMIT,
+        totalResults: response?.data?.count ?? 0,
+      };
+    },
```

Comment:
`request` is deprecated. `query` if possible, or use [`callApi`](https://github.com/ohcnetwork/care_fe/blob/develop/src/Utils/request/query.ts#L7) as an alternative.